Pod/Classes/SEGGoogleAnalyticsIntegration.m

```diff
@@ -223,6 +224,11 @@ - (NSDictionary *)setCustomDimensionsAndMetricsAndCampaignData:(NSDictionary *)p
         // kGAICampaignAdNetworkId
     }
 
+    if ([event isEqualToString:@"Deep Link Opened"]) {
+        [hit setCampaignParametersFromUrl:[properties valueForKey:@"url"]];
+        SEGLog(@"[hit setCampaignParametersFromUril: %@]", [properties valueForKey:@"url"]);
```

Comment:
`setCampaignParametersFromUril` should prob be `setCampaignParametersFromUril` - extra random "i" near the end. :)